### PR TITLE
Add man page

### PR DESCRIPTION
**Pull request by R1kaB3rN**
Adds a simple man page to the project that contains some example usages for users, which avoids the need to refer to the Github page for concrete examples. Targeted for system package installations, and for users who use ULWGL directly from the CLI instead from a client application.

The man page will be generated by [scdoc](https://git.sr.ht/~sircmpwn/scdoc), thus will be a build dependency.

### CODE REVIEW

File: ULWGL/ulwgl_consts.py [hunk 1]
```diff
@@ -26,16 +26,3 @@ class Color(Enum):
...
-  GAMEID= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= ulwgl-run /home/foo/example.exe -opengl
-  WINEPREFIX= GAMEID= PROTONPATH= ulwgl-run ""
-  WINEPREFIX= GAMEID= PROTONPATH= PROTON_VERB= ulwgl-run /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= STORE= ulwgl-run /home/foo/example.exe
-  ULWGL_LOG= GAMEID= ulwgl-run /home/foo/example.exe
-  ulwgl-run --config /home/foo/example.toml
-"""
```
Comment (loathingKernel) [inline ULWGL/ulwgl_consts.py:41]: My opinion is that this should not be removed. These are quick examples, manpage can also go in depth about various things about ULWGL as a whole on top of usage, such as information about the runtime, making is harder to quickly parse. IMHO each serve their own purpose despite the overlap.

File: Makefile.in [hunk 2]
```diff
@@ -7,6 +7,7 @@ PREFIX ?= /usr
 BINDIR := $(PREFIX)/bin
 LIBDIR := $(PREFIX)/lib
 DATADIR := $(PREFIX)/share
+MANDIR ?= $(PREFIX)/share/man
```
Comment (loathingKernel) [inline Makefile.in]: This should be tied to the `DATADIR` as `$(DATADIR)/man`

From FHS, both [`/usr/share/man`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html#usrsharemanManualPages) and [`/usr/local/share/man`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html#usrlocalshare1) are valid and they depend on the `DATADIR`

File: Makefile.in [hunk 3]
```diff
@@ -109,12 +110,18 @@ user-install:
 	$(info :: Installed under user-only location "$(DATADIR)/$(INSTALLDIR)")
 	$(info :: To run you need to make sure "$(BINDIR)" is in your PATH)
 
+.PHONY: docs
+ulwgl-docs-install:
+	$(info :: Installing man pages)
+	@mkdir -p $(DESTDIR)$(MANDIR)/man1
+	scdoc < docs/ulwgl.1.scd > $(OBJDIR)/ulwgl.1
+	install -m644 $(OBJDIR)/ulwgl.1 $(DESTDIR)$(MANDIR)/man1/ulwgl.1
```
Comment (loathingKernel) [inline Makefile.in]: This preferably should be two steps, one to build the manpage and a second step to install the manpage.

File: ULWGL/ulwgl_consts.py (hunk 1 above)
Comment (R1kaB3rN) [reply ULWGL/ulwgl_consts.py:41]: I actually don't share that opinion, and my original intention was actually a man page from the beginning. The usage was just a stand-in until there was man page created, which is now.

However, if this is really a desired feature for the minority of CLI users, I'll be willing to bring back the usage.

**Pull request merged**